doc/cephadm/octopus_gsg.rst

```diff
@@ -0,0 +1,262 @@
+.. _octopus_gsg:
+
+Installing Ceph Octopus
+=======================
+
+This procedure is a step-by-step walkthrough of the installation of a three-node Ceph clutser. If you have never installed Ceph before and you don't know where to begin, begin here.
+
+Removing an old cluster
+-----------------------
+
+If you mess up the installation at any point during the procedure, you can run this two-step procedure to return your system to the beginning state. If you mess up or you lose your place or you just want to start over from the beginning, now you know how.
+
+1. Run ``cephadm ls`` and make a note of the filesystem ID (fsid)::
+
+        [root@192-168-1-113 ~]# cephadm ls
+        [
+            {
+                "style": "cephadm:v1",
+                "name": "mon.192-168-1-113",
+                "fsid": "695cb1be-a0d8-11ea-89f0-48f17fce7093",
+                "enabled": true,
+                "state": "running",
+                "container_id": "9ea843bd19b47804c4dc1ee829b3a3d244afc37d4b9a737725bb957b0b64bc5b",
+                "container_image_name": "docker.io/ceph/ceph:v15",
+                "container_image_id": "4569944bb86c3f9b5286057a558a3f852156079f759c9734e54d4f64092be9fa",
+
+2. Use the command ``cephadm rm cluster`` to remove the cluster. Use the FSID that you made a note of in the previous step as the argument of the ``--fsid`` flag. Use the ``--force`` flag::
+
+        [root@192-168-1-113 ~]# cephadm rm-cluster --fsid 695cb1be-a0d8-11ea-89f0-48f17fce7093 --force
+
+Octopus Installation Procedure
+------------------------------
+1. The command ``cephadm bootstrap`` creates a new cluster. It must be run with the ``mon-ip`` flag. In this example, the ``--allow-fqdn-hostname`` flag has also been included::
+
+        [root@192-168-1-113 ~]# cephadm bootstrap --mon-ip 192.168.1.113 --allow-fqdn-hostname
```

Comment:
This output is really hard to read. It needs better indentation to separate your console from the output.